charts/dvpe-deployment-gloo/templates/deployment.yaml

```diff
@@ -86,10 +86,21 @@ spec:
           {{- if .Values.additionalparameters.secretsApplied }}
           {{- range $key, $value := .Values.additionalparameters.secrets }}
           - name: "{{$key}}"
+            valueFrom:
+              secretKeyRef: {{ $value.secretKeyRef }}
+          {{- end}}
+          {{- end}}
+            {{- if .Values.infrastructure }}
```

Comment:
you need to add your values to the values.yaml. Otherwise it is not documented

Reply:
Only those used in this github repo or also those used by the actual templates?